sycl/test-e2e/Basic/fpga_tests/fpga_io_pipes.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} -fsyntax-only -o %t.out
+// RUN: %{build} -fsyntax-only -Wno-error=unused-command-line-argument -o %t.out
```

Comment:
If we don't execute a test, then we should move it to `sycl/test`, i.e. `check-sycl`. From there we can decide if we want to also have a copy of it which we execute.

In this case, I object against the change and I would happily approve move of the test (except that I'm not a code owner :)). Feel free to send a separate PR, we should be able to merge it almost instantly